usaspending_api/api_contracts/contracts/award/idv/Funding.md

```diff
@@ -115,6 +115,19 @@ Returns File C financial data for an IDV (Indefinite Delivery Vehicle) award's d
                 }
             }
 
+# IDV Funding Roll-up [/api/v2/awards/idvs/funding-rollup/]
+
+This end point returns award metadata specifying the total transaction obligations, awarding agencies, and federal accounts.
+
+## POST
+
++ Request (application/json)
+    + Attributes (object)
+        + award_id: `TEST` (required, string)
```

Comment:
I think we'll want to change this to an actual IDV id for running compliance tests (confirm with backend)